symfony/console/4.2/bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+if (!class_exists(Application::class)) {
```

Comment:
the autoload has not yet been called at this point so this condition is always false and always throw the exception even if `framework-bundle` is installed

Reply:
Maybe because there is no `require __DIR__.'/../vendor/autoload.php';` in this file